pkg/scheduler/core/generic_scheduler.go

```diff
@@ -473,12 +473,10 @@ func (g *genericScheduler) findNodesThatFit(pod *v1.Pod, nodes []*v1.Node) ([]*v
 				return
 			}
 			if fits {
+				filtered[length] = g.nodeInfoSnapshot.NodeInfoMap[nodeName].Node()
```

Comment:
length is used before declaration.  If I understand correctly, you want to use filteredLen?